src/Engine/ProtoCore/Parser/AssociativeAST.cs

```diff
@@ -1287,6 +1289,26 @@ public MethodAttributes(bool hiddenInLibrary = false, string msg = "")
         }
     }
 
+    public class ExternalAttributes
```

Comment:
I remember we did some support on attributes from C# and integrating them to DS, Just want to clarify if that is indeed the case, do you think it wont have any overlap with ExternalAttributes?